dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.h

```diff
@@ -231,10 +231,11 @@ class OpenDDS_Rtps_Udp_Export RtpsUdpDataLink : public DataLink, public virtual
   GuidPrefix_t local_prefix_;
 
   struct RemoteInfo {
-    RemoteInfo() : addr_(), requires_inline_qos_(false) {}
-    RemoteInfo(const ACE_INET_Addr& addr, bool iqos)
-      : addr_(addr), requires_inline_qos_(iqos) {}
-    ACE_INET_Addr addr_;
+    RemoteInfo() : narrow_addr_(), wide_addr_(), requires_inline_qos_(false) {}
+    RemoteInfo(const ACE_INET_Addr& narrow_addr, const ACE_INET_Addr& wide_addr, bool iqos)
+      : narrow_addr_(narrow_addr), wide_addr_(wide_addr), requires_inline_qos_(iqos) {}
+    ACE_INET_Addr narrow_addr_;
+    ACE_INET_Addr wide_addr_;
```

Comment:
`unicast_addr_` and `multicast_addr_`?

Reply:
I think the reason I didn't just say unicast and multicast is that when multicast is disabled they're both the unicast address. Also, if for whatever reason we only have multicast addresses to chose from, the unicast would be multicast. So you could be debugging and see that multicast_addr_ was actually a unicast address and you might think it was a bug or something. But maybe narrow and wide are just as confusing...